iOS/ViewControllers/Account/AppearanceSettingsViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AppearanceSettingsViewController: UITableViewController {
+
+    @available(iOS 13.0, *)
+    private var theme: Theme {
+        get {
+            return UserDefaults.standard.theme
+        }
+        set {
+            UserDefaults.standard.theme = newValue
+            self.configureStyle(for: newValue)
+        }
+    }
+
+    @available(iOS 13.0, *)
+    private func configureStyle(for theme: Theme) {
+        UIApplication.shared.windows.forEach { window in
+            window.overrideUserInterfaceStyle = theme.userInterfaceStyle
+        }
+    }
```

Comment:
Referring to `UIApplication.shared.windows` is not considered as a best practice and should be avoided whenever possible (referring to `UIApplication` in general).

In an earlier commit, you already utilized the KVO-pattern to observe the changes to the `Theme` value in `UserDefaults` in this view controller. However, to change the appearance of all windows, each `SceneDelegate` instance should start the observation of this value.